docs/integrations/destinations/databricks.md

```diff
@@ -0,0 +1,101 @@
+# Databricks
+
+## Overview
+
+This destination syncs data to Databricks cluster. Each stream is written to its own table.
+
+This connector requires a JDBC driver to connect to Databricks cluster. The driver is developed by Simba. Before using the driver and the connector, you must agree to the [JDBC ODBC driver license](https://databricks.com/jdbc-odbc-driver-license). This means that you can only use this connector to connector third party applications to Apache Spark SQL within a Databricks offering using the ODBC and/or JDBC protocols.
+
+## Sync Mode
+
+| Feature | Support | Notes |
+| :--- | :---: | :--- |
+| Full Refresh Sync | ✅ | Warning: this mode deletes all previously synced data in the configured bucket path. |
+| Incremental - Append Sync | ✅ | |
```

Comment:
does it also support incremental dedupe?

Reply:
Currently no.  This implementation uses the recommended `COPY INTO` command, which is faster but only supports insertion. We can support dedup by using the `MERGE` command. It is slower, but it does support update and deletion.

Reference (from the `Ingesting data into Delta` Databricks sent us):

> ## Copy Into SQL command VS Merge SQL command
> - The Copy Into SQL command is designed to support high performence inserts (new data) into Databricks Delta. It does not support Updates and Deletes
> - Merge SQL command supports inserts updates and deletes. It is slower that the Copy Into command for inserting data into Delta.

I can address this in a follow-up PR. Created an issue here: #6042.

Reply:
let's add it in the doc then? 
```suggestion
| Incremental - Append Sync | ✅ | |
| Incremental - Dedupe Sync | ❌ | |
```